spec/helpers/comments_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the CommentsHelper. For example:
+#
+# describe CommentsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```

Comment:
- [ Required ] I think it would be nice if you remove any piece of commented code in this file, commented code usually causes confusion and is considered as a sign of code that is not clean. It also makes your project look unprofessional, please rectify this.